export/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ExportAdapter.java

```diff
@@ -79,22 +83,18 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
       System.exit(1);
     } else if (isStowRs) {
       // STOW-RS sender.
-      //
-      // DicomWeb client for sink of DICOM.
-      // Use application default credentials for HTTP requests of DICOM sink, if specified by flag.
-      HttpRequestFactory requestFactory = null;
-      if (flags.useGcpApplicationDefaultCredentials) {
-        requestFactory = createHttpRequestFactory(credentials);
-      } else {
-        requestFactory = createHttpRequestFactory(null);
-      }
-      DicomWebClient exportDicomWebClient =
-          new DicomWebClient(requestFactory, flags.peerDicomwebAddr);
+      boolean isLegacyAdress = flags.peerDicomwebAddress.isEmpty();
+      String peerDicomwebAddress =
+          isLegacyAdress ? flags.peerDicomwebAddr : flags.peerDicomwebAddress;
+      String peerDicomwebStowpath = isLegacyAdress ? flags.peerDicomwebStowPath : "studies";
+      IDicomWebClient exportDicomWebClient =
```

Comment:
Would any post request succeed without this flag set to true? How would we get any credentials for the request?

Reply:
I assumed sending request without auth header was the intention here, but after looking deeper it doesn't seem to be right.

Though I'm also not sure under which conditions this flag was useful.
credentials are obtained as:

```
GoogleCredentials credentials = GoogleCredentials.getApplicationDefault();
if (!flags.oauthScopes.isEmpty()) {
      credentials = credentials.createScoped(Arrays.asList(flags.oauthScopes.split(",")));
}
```

The flag was used  like this:

```
// DicomWeb client for sink of DICOM.
// Use application default credentials for HTTP requests of DICOM sink, if specified by flag.
      HttpRequestFactory requestFactory = null;
      if (flags.useGcpApplicationDefaultCredentials) {
        requestFactory = createHttpRequestFactory(credentials);
      } else {
        requestFactory = createHttpRequestFactory(null);
      }
      DicomWebClient exportDicomWebClient =
          new DicomWebClient(requestFactory, flags.peerDicomwebAddr);
```

... So, assuming comments here are correct, either way it's application default credentials, with scopes being the only difference.

Can I just remove or ignore this flag?

Reply:
From the comment on Flags.java: 
"If true, uses GCP Application Credentials to when sending HTTP requests to peer. This is useful if the peer DICOMWeb endpoint is running in GCP."
So I realized that this is for non-GCP dicomWeb backends. So it's fine to leave as-is.